clash-ffi/clash-ffi.cabal

```diff
@@ -1,7 +1,7 @@
 cabal-version:      2.2
 
 name:               clash-ffi
-version:            1.5.0
+version:            1.6.0
```

Comment:
`clash-ffi` is locked to Clash's versioning (for now). This should have been 1.7.0. (Note that odd numbers indicate development versions, and even numbers released versions - similar to GHC.)